packaging/debian/control

```diff
@@ -29,3 +29,9 @@ Architecture: any
 Depends: ${misc:Depends}, nvidia-container-toolkit-base (= @VERSION@)
 Description: NVIDIA Container Toolkit Operator Extensions
  Provides tools for using the NVIDIA Container Toolkit with the GPU Operator
+
+Package: nvidia-container-toolkit-cdi-refresh
```

Comment:
As I called out in a previous comment, we don't have the same package in the RPM package spec. Could we discuss why these are not just part of the `-base` package?